crates/qt-build-utils/src/parse_cflags.rs

```diff
@@ -196,6 +196,7 @@ pub(crate) fn parse_libs_cflags(name: &str, link_args: &[u8], _builder: &mut cc:
                         } else {
                             match extract_lib_from_filename(target, &file_name) {
                                 Some(lib_basename) => {
+                                    println!("cargo:rustc-link-search=lib_from_filename :",);
```

Comment:
trace line?

Reply:
Yes, but I thought that it was useful. I'll remove it
```suggestion
```